client/src/components/RegistrationForm/NewRegistration.tsx

```diff
@@ -61,8 +62,8 @@ export const NewRegistration: React.FC<NewRegistrationProps> = ({
   const {
     config: writeSubmitRegistrationConfig
   } = usePrepareContractWrite({
-    address: rampAddress,
-    abi: rampAbi,
+    address: rampAddress ?? undefined,
+    abi: rampAbi ?? undefined,
```

Comment:
Need a better solution for the typing mismatch here. I've left all sc writes like this for now until we can address them all